docs/pages/database-access/guides/sql-server-ad-pkinit.mdx

```diff
@@ -0,0 +1,387 @@
+---
+title: Database Access with Microsoft SQL Server with Active Directory PKINIT authentication (Preview)
+description: How to configure Teleport Database Access with Microsoft SQL Server with Active Directory PKINIT authentication.
+---
+
+<Admonition type="warning" title="Preview">
+  Database Access for Microsoft SQL Server is currently in Preview mode.
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Set up access to SQL Server using Active Directory PKINIT authentication.
+- Connect to SQL Server through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access SQL Server Self-Hosted](../../../img/database-access/guides/sqlserver_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access SQL Server Cloud](../../../img/database-access/guides/sqlserver_cloud.png)
+</ScopedBlock>
+
+This guide will focus on SQL Servers using self-hosted Active Directory
+authentication.
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A SQL Server database with Active Directory authentication enabled.
+- A SQL Server network listener configured with a Certificate using Subject Alternative Names
```

Comment:
```suggestion
- A SQL Server network listener configured with a certificate using Subject Alternative Names.
```
- "Certificate" is not a proper noun
- Using consistent punctuation in this list